routes/v1.js

```diff
@@ -193,7 +193,7 @@ function addRoutes(app, peliasConfig) {
   app.get ( base + 'autocomplete', routers.autocomplete );
   app.get ( base + 'search',       routers.search );
   app.post( base + 'search',       routers.search );
-  app.get ( base + 'beta/component',    routers.component );
+  app.get ( base + 'search/structured',    routers.structured );
```

Comment:
since the spacing here attempts to line up the `routers` column maybe moving all of them over a bit would be nicer